src/components/sdk/VideoTileGrid/index.tsx

```diff
@@ -38,11 +38,14 @@ interface Props extends BaseProps {
   noRemoteVideoView?: React.ReactNode;
   /** The layout of the grid. */
   layout?: Layout;
+  /** add local user name in localtile */
```

Comment:
For the consistency, can we capitalize the start of this statement, also, should we say "The local video tile nameplate name. Default is 'Me'." ?

Reply:
Sure.., will change it..